bzt/modules/jmeter.py

```diff
@@ -358,23 +407,29 @@ def __get_modified_jmx(self, original, load):
 
         self.__apply_modifications(jmx)
 
+        rename_threads = self.get_scenario().get("rename-threads", True)
```

Comment:
I see no mentions on this option in documentation. And it's module-level, not scenario-level option.